transport/http_transport.go

```diff
@@ -469,7 +502,16 @@ func waitForConnect(ctx context.Context, address string, cfg *tls.Config) error
 			errs = append(errs, ctx.Err().Error())
 			return errors.New(strings.Join(errs, "\n"))
 		}
-		time.Sleep(time.Millisecond * 100)
+		wait(time.Millisecond * 100)
```

Comment:
Can you explain why `time.Sleep` isn't suitable here.

Reply:
@thrawn01 we can use sleep. This is just a preference.

Reply:
Let's use sleep please.